Week-5/fail.test.js

```diff
@@ -0,0 +1,13 @@
+const mathOperations = require("./mathOps");
+
+describe("Maths operation test addition, diff, product", () => {
+  test("adding 1 and 2 should return 3", () => {
```

Comment:
@rajeshvlega9 It looks good to me. But for checking failing scenarios 
add more like checking for no arguments passed to the method, what if only one argument is passed. Etc